CONTRIBUTORS.md

```diff
@@ -128,6 +129,7 @@ Created by running `./build-support/bin/contributors.sh`.
 + Roger Jiang
 + Ryan Williams
 + Sameer Brenn
```

Comment:
Sorry, missed this until just now. 'Sameer Brenn' is the correct, new name. Thanks!